src/hooks/useContentProtection.ts

```diff
@@ -4,7 +4,8 @@ import { useConfigStore } from '#src/stores/ConfigStore';
 import type { GetPlaylistParams } from '#types/playlist';
 import type { GetMediaParams } from '#types/media';
 import AccountController from '#src/stores/AccountController';
-import EntitlementService from '#src/services/entitlement.service';
+import GenericEntitlementService from '#src/services/generic_entitlement.service';
+import JWPEntitlementService from '#src/services/jwp_entitlement.service';
```

Comment:
Suggestion (not important): `GenericEntitlementService` -> `SelfHostedEntitlementService`

I suggest keeping the file name convention the same for now -> `jwpEntitlement.service.ts`. I prefer renaming all services to PascalCase (because they are actual classes now), but we can do that later.

Reply:
Yeah, we should definitely rename them. vs-code icons even treat them as angular services 😆 

<img width="472" alt="Screenshot 2023-11-14 at 14 02 59" src="https://github.com/jwplayer/ott-web-app/assets/27254986/c9d0410c-9d21-4ec5-aedc-e0a65ad57d5f">